### PR TITLE
Unpin ansible-lint

### LINKED ISSUE #69: Molecule builds currently broken
# 🐛 Summary #

[Version 5 of `ansible-lint` has been released](https://github.com/ansible-community/ansible-lint/releases/tag/v5.0.0), and it contains some breaking changes that have yet to shake out.

See:
* https://github.com/ansible-community/ansible-lint/discussions/1150
* https://github.com/ansible-community/ansible-lint/issues/1329
* https://github.com/ansible-community/ansible-lint/issues/1344

## To reproduce ##

1. Check out the code for any cisagov Ansible role that is based on [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role) or [cisagov/skeleton-ansible-role-with-test-user](https://github.com/cisagov/skeleton-ansible-role-with-test-user).
2. Run `pip install --upgrade -r requirements-dev.txt`.
3. Run `molecule test` and watch it fail.

## Expected behavior ##

`molecule test` should complete successfully for any cisagov Ansible role that is based on [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role) or [cisagov/skeleton-ansible-role-with-test-user](https://github.com/cisagov/skeleton-ansible-role-with-test-user).

## Any helpful log output or screenshots ##

For an example of this bug in an APB build, see [here](https://github.com/cisagov/ansible-role-nvme/actions/runs/568458801).

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request:
- Unpins `ansible-lint` (it was previously restricted to be `<5`)
- Adds a configuration file for `ansible-lint`
- Adds a `cisagov` namespace to the role metadata

## 💭 Motivation and context ##

We have been avoiding version 5 of `ansible-lint` for a while because it introduced some breaking changes.  The dust has now settled, and I have a way forward (thanks to @geerlingguy).

Resolves #69.

## 🧪 Testing ##

All `pre-commit` hooks and `molecule` tests pass.

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Looks good- thanks for delving into this!  🚫📌

File: requirements-test.txt [hunk 1]
```diff
@@ -5,11 +5,7 @@
 # Kali Linux, so it makes sense to force the installation of Ansible
 # 2.10 or newer.
 ansible>=2.10,<5
-# ansible-lint has a number of breaking changes with the update to v5 as we have
-# documented in https://github.com/cisagov/skeleton-ansible-role/issues/69
-# We can unpin once everything with the v5 update has shaken out and we have a
-# clear path forward.
-ansible-lint<5
+ansible-lint
```
Comment (mcdonnnj) [inline requirements-test.txt]: Some version pinning to reflect the version these changes require while still keeping us from getting blind-sided by possible future changes.

```suggestion
ansible-lint>=5,<6
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Thanks for keeping an eye on this while we waited! I just had one request around version pinning given our previous experience.

**Review by mcdonnnj — APPROVED** [review]
:godmode:

**Pull request merged**